src/filter-dropdown.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+export default class FilterDrop extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {toggle: false}
+    this.toggler = this.toggler.bind(this)
+  }
+  toggler() {
+    this.setState({toggle: !this.state.toggle})
+  }
+  render() {
+    const topics = []
+    const drop = this.state.toggle ? 'd-block' : 'd-none'
+    return (
+      <div className="dropdown">
+        <button onClick={this.toggler} className="btn btn-primary mt-3 ml-1">Filter By:</button>
+        <div>
+          {this.props.cards
```

Comment:
Compute your topic list in your card list component so `FilterDrop` doesn't need to figure it out. Remember, "be conservative in what you send".